ipns/ipns.go

```diff
@@ -120,6 +122,7 @@ func (m *Manager) Publish(pth path.Path, keyID string) {
 			cancel, ok := m.ctxs[keyID]
 			m.ctxsLock.Unlock()
 			if ok {
+				log.Debugf("success path %s: for key %s", pth, keyID)
```

Comment:
Success is already logged below in `publishUnsafe`

Reply:
yeah, these are all the tidy things to wrap up after choosing the right strategy. taking out of wip here in a minute